modules/ztp-telco-ran-software-versions.adoc

```diff
@@ -24,7 +24,7 @@ The Red Hat Telco Radio Access Network (RAN) version {product-version} solution
 |2.7
 
 |{gitops-title}
-|1.6
```

Comment:
I know its an example, but does the following example need to be updated?

![Screenshot from 2023-09-26 09-50-04](https://github.com/openshift/openshift-docs/assets/57954076/cfccf1a3-b062-4178-9fc5-7447f8737ebc)

See [Preparing to download the OpenShift Container Platform images](https://65164--docspreview.netlify.app/openshift-enterprise/latest/scalability_and_performance/ztp_far_edge/ztp-precaching-tool#ztp-preparing-ocp-images_pre-caching)

Reply:
Probably ok for example output, I don't think it will effect anything